ucsb-cs156-f23/proj-happycows-f23-5pm-4
# FEATURE: Frontend for default values for Commonds fields in application.properties / env variables

## Overview

Show default values set in application.properties / .env on CommonsCreate page form.

## Future Possibilities (Optional)

Some possible points could be:
- Possibly include some sort of validation for default values? (another student mentioned this in the happycows slack channel)

## Validation (Optional)

1. Download this branch.
2. Run project
3. Navigate to Commons Create page
4. Modify application.properties / .env
5. Reload and notice that default values have changed
6. Submit and note that the default values are passed into the new commons that was just created

## Tests


## Linked Issues

Closes #12
